packages/get-credentials/src/get-credentials.js

```diff
@@ -0,0 +1,60 @@
+import dotenv from 'dotenv'
+import path from 'path'
+
+import { homepage } from '../package.json'
+
+export default function getCredentials (projectKey) {
+  if (!projectKey)
+    return Promise.reject([new Error('Project Key is needed')])
```

Comment:
Throwing it? The function should always return a promise right?

It's in an array because the function can return multiple errors, if it were to only sometimes return an array that would make it harder to use I think.